modules/superset/tests/superset_with_ingress/main.tf

```diff
@@ -0,0 +1,34 @@
+module "test" {
+  source = "../../"
+
+  name               = "superset"
+  namespace          = "superset"
+  cluster_name       = "buycycle-cluster"
+  supeset_secret_key = "sdgiadgiagdiagdi"
+
+  postgress_config = {
+    create = false
+    host   = "postgress-aws-hostname"
+    port   = "5432"
+    user   = "superset"
+    pass   = "superset"
+    name   = "superset"
+  }
+
+  redis_config = {
+    create = false,
+    host   = "external-redis-hostname",
+    port   = 6379
+  }
+
+  alarms = {
+    enabled   = false
+    sns_topic = ""
+  }
+
+  ingress_config = {
+    create = true
+    hosts  = ["superset.test.dasmeta.com"]
```

Comment:
Do we need to have hosts? won't one be enough?

Reply:
Depending on the case, one should generally be enough